src/components/AddAnnonce/Infos/index.jsx

```diff
@@ -0,0 +1,225 @@
+import PropTypes from 'prop-types';
+import SInfo from './style';
+
+const Infos = ({
+  inputFurnished,
+  inputSurface,
+  inputAnimals,
+  inputType,
+  inputEnergyClass,
+  inputGes,
+  inputRooms,
+  setType,
+  setSurface,
+  setFurnished,
+  setRooms,
+  setAnimals,
+  setEnergyClass,
+  setGes,
+}) => {
+  const handleChangeType = (evt) => {
+    setType(evt.target.value);
+  };
+  const handleChangeSurface = (evt) => {
+    setSurface(evt.target.value);
+  };
+
+  const handleChangeFurnished = (evt) => {
+    setFurnished(evt.target.value);
+  };
+
+  const handleChangeAnimals = (evt) => {
+    setAnimals(evt.target.value);
+  };
+
+  const handleChangeEnergyClass = (evt) => {
+    setEnergyClass(evt.target.value);
+  };
+  const handleChangeGes = (evt) => {
+    setGes(evt.target.value);
+  };
+
+  const seeIfSup = () => {
+    if (inputRooms > 0) {
+      setRooms(inputRooms - 1);
+    } else {
+      setRooms(inputRooms);
+    }
+  };
+  return (
+    <SInfo>
+      <div className="partContainer">
+        <div className="box1">
+          <div className="boxPlaceType">
+            <h2>Type de logement :</h2>
+            <div className="radioContainer">
+              <label htmlFor="radio_1">
+                <input
+                  id="radio_1"
+                  type="radio"
+                  name="place-type"
+                  value="Maison"
+                  onClick={handleChangeType}
+                  checked={inputType === 'Maison'}
+                />
+                Maison
+              </label>
+              <label htmlFor="radio_2">
+                <input
+                  id="radio_2"
+                  type="radio"
+                  name="place-type"
+                  value="Appartement"
+                  onClick={handleChangeType}
+                  checked={inputType === 'Appartement'}
+                />
+                Appartement
+              </label>
+            </div>
+          </div>
+          <div className="boxFurnished">
+            <h2>Meublé :</h2>
+            <select value={inputFurnished} onChange={handleChangeFurnished}>
+              <option selected value="">
+                {' '}
```

Comment:
Same here (useless code) "{' '}" 😒

Reply:
Impossible to change with the custom repo